File: bindings/go/binding_test.go
package tree_sitter_bicep_test

import (
	"testing"

	tree_sitter "github.com/tree-sitter/go-tree-sitter"
	tree_sitter_bicep "github.com/tree-sitter-grammars/tree-sitter-bicep/bindings/go"
)

func TestCanLoadBicepGrammar(t *testing.T) {
	language := tree_sitter.NewLanguage(tree_sitter_bicep.LanguageBicep())
	if language == nil {
		t.Errorf("Error loading Bicep grammar")
	}
}

func TestCanLoadBicepParamsGrammar(t *testing.T) {
	language := tree_sitter.NewLanguage(tree_sitter_bicep.LanguageBicepParams())
	if language == nil {
		t.Errorf("Error loading Bicep Params grammar")
	}
}

File: bindings/node/binding_test.js
const assert = require("node:assert");
const { test } = require("node:test");

const Parser = require("tree-sitter");

test("can load Bicep grammar", () => {
  const parser = new Parser();
  assert.doesNotThrow(() => parser.setLanguage(require(".").bicep));
});

test("can load Bicep Params grammar", () => {
  const parser = new Parser();
  assert.doesNotThrow(() => parser.setLanguage(require(".").bicep_params));
});
